Kiosk/App/Networking/ArtsyAPI.swift

```diff
@@ -50,15 +50,18 @@ enum ArtsyAuthenticatedAPI {
 
 extension ArtsyAPI : TargetType, ArtsyAPIType {
 
-    public var parameterEncoding: ParameterEncoding {
-        return JSONEncoding.default
+    var headers: [String : String]? {
+        return nil
     }
 
-    public var task: Task {
-        return .request
+    var task: Task {
+        if let requestParameters = parameters {
+            return .requestParameters(parameters: requestParameters, encoding: JSONEncoding.default)
```

Comment:
This is one other thing that changed with Moya 9 – looks like a longstanding bug that happened to be fixed that this code relied on. 

Old behaviour: encoding POST requests with the JSON encoder as specified, otherwise encode them in the URL. Pretty easy to work around: https://github.com/artsy/eidolon/pull/678/files#diff-2e89f1c7f10111aa886cbf6eb3922b40R273